build-system/compile/pre-closure-babel.js

```diff
@@ -16,29 +16,14 @@
 'use strict';
 
 const argv = require('minimist')(process.argv.slice(2));
-const babel = require('@babel/core');
 const conf = require('./build.conf');
+const fs = require('fs');
 const globby = require('globby');
-const path = require('path');
+const gulpBabel = require('gulp-babel');
+const gulpCache = require('gulp-cache');
 const through = require('through2');
 const {BABEL_SRC_GLOBS, THIRD_PARTY_TRANSFORM_GLOBS} = require('./sources');
 
-const ROOT_DIR = path.resolve(__dirname, '../../');
-
-/**
- * Files on which to run pre-closure babel transforms.
- *
- * @private @const {!Array<string>}
- */
-const filesToTransform = getFilesToTransform();
```

Comment:
`getFilesToTransform()` will get called O(N) times by moving it inside `preClosureBabel()`. Why not call it here once and for all at the module level?

Reply:
Reverted.